src/navigators/SendPaymentNavigator.tsx

```diff
@@ -0,0 +1,72 @@
+/* eslint-disable react/no-unstable-nested-components */
+import { createNativeStackNavigator } from "@react-navigation/native-stack";
+import CustomNavigationHeader from "components/CustomNavigationHeader";
+import { SendHome } from "components/screens/SendScreen";
+import { 
+  MemoScreen,
+  TransactionDetailScreen,
+  TransactionFeeScreen,
+  TransactionTimeoutScreen,
+  TransactionValueScreen 
+} from "components/screens/SendScreen/screens";
+import { SEND_PAYMENT_ROUTES, SendPaymentStackParamList } from "config/routes";
+import useAppTranslation from "hooks/useAppTranslation";
+import React from "react";
+
+const SendPaymentStack =
+  createNativeStackNavigator<SendPaymentStackParamList>();
+
+export const SendPaymentStackNavigator = () => {
+  const { t } = useAppTranslation();
+
+  return (
+    <SendPaymentStack.Navigator
+      screenOptions={{
+        header: (props) => <CustomNavigationHeader {...props} />,
+      }}
+    >
+      <SendPaymentStack.Screen
+        name={SEND_PAYMENT_ROUTES.SEND_PAYMENT_SCREEN}
+        component={SendHome}
+        options={{
+          headerTitle: t("sendPaymentScreen.title"),
+        }}
+      />
+      <SendPaymentStack.Screen
+        name={SEND_PAYMENT_ROUTES.TRANSACTION_DETAIL_SCREEN}
```

Comment:
```suggestion
        name={SEND_PAYMENT_ROUTES.TRANSACTION_DETAILS_SCREEN}
```

Reply:
I think we should actually use a different name here, see my other comment [here](https://github.com/stellar/freighter-mobile/pull/112/files#r2068840473)